contracts/modules/licensing/PILicenseTemplate.sol

```diff
@@ -22,7 +25,9 @@ contract PILicenseTemplate is
     BaseLicenseTemplateUpgradeable,
     IPILicenseTemplate,
     LicensorApprovalChecker,
-    ReentrancyGuardUpgradeable
+    AccessManagedUpgradeable,
```

Comment:
We might not need this `AccessManagedUpgradeable`, since this contract doesn't have any function under the governance. 
If somehow we need to add or update a function to be under governance we will upgrade this contract.

Reply:
It's needed for `restricted` in `_authorizeUpgrade`, which makes upgrade only possible by the protocol governance.

```solidity
/// @dev Hook to authorize the upgrade according to UUPSUpgradeable
/// @param newImplementation The address of the new implementation
function _authorizeUpgrade(address newImplementation) internal override restricted {}
```

Reply:
I see. then we need it indeed.